9-react.ts+TDD/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "9-react",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.ts",
+  "scripts": {
+    "test": "jest",
+    "serve": "webpack serve",
+    "build": "webpack --mode=production",
+    "start": "webpack serve"
+  },
+  "dependencies": {
+    "@babel/preset-env": "^7.14.4",
+    "@babel/preset-react": "^7.13.13",
+    "@testing-library/jest-dom": "^5.12.0",
+    "@types/enzyme": "^3.10.8",
```

Comment:
read about the difference between `dependencies` and `devDependencies` and re-organize your dep list.